pkg/integration/cedar_test.go

```diff
@@ -26,9 +26,9 @@ func (s *IntegrationTestSuite) TestCEDARConnection() {
 	s.NoError(err)
 	rr := httptest.NewRecorder()
 
-	cedarClient := cedar.NewTranslatedClient(
+	cedarClient := cedareasi.NewTranslatedClient(
```

Comment:
we need to add tests for cedar ldap as well.

Reply:
The cedar tests we have already don't actually hit cedar at all, they just test the methods we use to validate submission of an intake to cedar. We don't have any of those methods in LDAP. What tests, then, should I write for this?